src/main/java/com/opentok/SessionProperties.java

```diff
@@ -120,19 +117,46 @@ public Builder archiveMode(ArchiveMode archiveMode) {
             return this;
         }
 
+        /**
+         * End-to-end encryption (or E2EE) allows application developers to encrypt media in routed sessions
+         * from client to client. Media is already encrypted P2P, client to client, through WebRTC protocols
+         * when using a relayed session. This feature adds an encryption layer by encrypting the media payload at the
+         * client so that it will remain encrypted through the media server when routing media to other clients.
+         * Therefore, you must also set {@link #mediaMode(MediaMode)} to {@linkplain MediaMode#ROUTED} when
```

Comment:
Let's just link to the docs at https://tokbox.com/developer/guides/end-to-end-encryption (instead of describing the feature here).

```suggestion
         * Enables <a href="https://tokbox.com/developer/guides/end-to-end-encryption">end-to-end encryption</a> for a routed session.
         * You must also set {@link #mediaMode(MediaMode)} to {@linkplain MediaMode#ROUTED} when
```